roles/openshift-applier/README.md

```diff
@@ -119,15 +121,18 @@ openshift_cluster_content:
 
 ### Filtering content based on tags
 
-The `openshift-applier` supports the use of tags in the inventory (see example above) to allow for filtering which content should be processed and not. The `filter_tags` variable/fact takes a comma separated list of tags that will be processed and only content/content_dir with matching tags will be applied. 
+The `openshift-applier` supports the use of tags in the inventory (see example above) to allow for filtering which content should be processed and not. The `filter_tags` variable/fact takes a comma separated list of tags that will be processed and only content/content_dir with matching tags will be applied.
 
 **_NOTE:_** Entries in the inventory without tags will not be procssed when a valid list is supplied with the `filter_tags` option.
 
 ```
 filter_tags=tag1,tag2
 
-``` 
+```
+
+### Deprovisioning
 
+The `openshift-applier` role also supports deprovisioning of resources using `file_action: delete` or `template_action: delete`. This can be done either on individual files, if you want to continually ensure objects remain deleted, or it may be done globally using `provision: false`. Using `provision: false` essentially acts like a big 'undo' button, re-running all files and templates through `oc delete -f <resources>`. This can be useful when you want to do a full cleanup to ensure the integrity of you IaC repo, or for simple cleanup while testing changes.
```

Comment:
I'd like to see the `provision: false` be called out in a new paragraph as it seems like a major feature. Including an example on how to run it would also be great.